config/webpack.config.js

```diff
@@ -368,6 +369,9 @@ module.exports = async (env = {}) => {
       noParse: [/mapbox\/vendor\/promise.js$/],
     },
     resolve: {
+      alias: {
+        '~': path.join(CWD, 'src'),
+      },
```

Comment:
This shouldn't be needed, our babel config already resolves `~`
https://github.com/department-of-veterans-affairs/vets-website/blob/main/.babelrc#L60

Reply:
Maybe I was getting ahead of myself - I was under the impression that we'd be using Typescript instead of Babel, sooner than later, and then this would be needed to support the paths. Good call out - I'll remove for now, and wait until we're ready for TS.

Reply:
updated